src/io/flutter/run/daemon/DeviceService.java

```diff
@@ -225,6 +226,8 @@ private DeviceDaemon chooseNextDaemon(Refreshable.Request<DeviceDaemon> request)
     emulatorManager.refresh();
 
     try {
+      // Wait for indexing to finish before starting daemon process
```

Comment:
nit: end comment with a period.
Do we have an idea why starting the daemon service in dumb mode causes problems?
If is worth commenting on it here as it is a little surprising that we have to wait for smart mode.